src/pages/LoginPage/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react';
+import { Redirect } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+
+import { useSession, useDispatch } from 'hooks';
+import { login } from 'state/actions/userActions';
+import routes from 'constants/routesPaths';
+import { APP_TITLE } from 'constants/constants';
+
+import LoginForm from 'components/user/LoginForm';
+import SocialMediaContainer from 'components/common/SocialMediaContainer';
+
+import { ReactComponent as Smilies } from 'assets/smilies.svg';
+
+const LoginPage = () => {
+  const { authenticated } = useSession();
+  const loginRequest = useDispatch(login);
+
+  if (authenticated) {
+    return <Redirect to={routes.index} />;
+  }
+
+  return (
+    <div className="login-page-container">
+      <div className="info-text-container">
```

Comment:
WDYTA moving this entire `.info-text-container` div to a new component (maybe inside de `LoginPage` folder)? This can be just a vanilla component and you'll reduce file complexity and size

You can probably do the same with the `.links-container` div below.

Reply:
As those two potential components don't have any particular logic and we haven't defined the Layout component yet, I'd rather let that html as it is for the moment. If different pages start to share the same HTML structure in the future, I would re-think this extraction, WDYTA it?

Reply:
Yeah, sounds good 😌